### PR TITLE
HMPOPR-84 - refactored tests to use page objects.

### PR DESCRIPTION
**Pull request by joefitter**
- As we add more journeys, steps will be shared. Page objects are a way of stopping code duplication

### CODE REVIEW

**Comment by joefitter** [comment]
@JoeChapman @bendiggle @easternbloc

**Comment by bendiggle** [comment]
@joefitter You're a wizard! LGTM
giphy

File: apps/enquiries/translations/src/en/fields.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "enquiry": {
+  "enquiry-type": {
     "legend": "What would you like to do?",
     "options": {
       "track": "Track a passport application",
```
Comment (JoeChapman) [inline apps/enquiries/translations/src/en/fields.json:5]: @joefitter If I were to be pedantic, I would say that the refactoring of the `enquiry` to `enquiry-type` should not be in this PR for page objects

**Comment by bendiggle** [comment]
@JoeChapman @easternbloc

File: apps/enquiries/translations/src/en/fields.json (hunk 1 above)
Comment (joefitter) [inline apps/enquiries/translations/src/en/fields.json:5]: I'm just gunna leave this here... :fu:

**Comment by joefitter** [comment]
😻

**Pull request merged**